xblock/core.py

```diff
@@ -202,34 +802,18 @@ def __init__(self, runtime, field_data=None, scope_ids=UNSET, *args, **kwargs):
         if scope_ids is UNSET:
             raise TypeError('scope_ids are required')
 
-        # Provide backwards compatibility for external access through _field_data
-        super().__init__(runtime=runtime, scope_ids=scope_ids, field_data=field_data, *args, **kwargs)
-
-    @property
-    def usage_key(self):
```

Comment:
usage_key and context_key are moved to Blocklike so that XBlockAsides can use them too.